src/libfuncs/circuit.rs

```diff
@@ -1382,12 +1382,12 @@ mod test {
             }
         );
 
-        dbg!(Felt::from_raw([
+        Felt::from_raw([
             576460752303419696,
             18446744073709551615,
             18446744073709551615,
-            18446744073709551393
-        ]));
+            18446744073709551393,
+        ]);
```

Comment:
Oh, I think is accidentally left this debug statement when I implemented circuits. It should be safe to remove (the whole statement).